docs/pages/links.md

```diff
@@ -198,16 +205,16 @@ variation_groups:
           appear in the following style and include both the original link text
           as well as a shortened URL.
         variation_code_snippet: Here's the <a href="#">link style</a> when printed.
-        variation_code_snippet_rendered: "<p>Here's the <a
-          href=\"https://consumerfinance.gov/about-us/blog\"
-          style=\"font-weight: 500;\">link style<span style=\"border-bottom: 1px
-          solid #ffffff; font-weight: 300;\">
-          (cfpb.gov/about-us/blog)</span></a> when printed.</p>"
+        variation_code_snippet_rendered: '<p>Here''s the <a
```

Comment:
These single quotes look iffy but everything is rendering as expected locally, so I'm not going to make any manual adjustments.